profiler/MAPL_Profiler.F90

```diff
@@ -39,5 +39,15 @@ module mapl_Profiler
    use mapl_SeparatorColumn
    
    implicit none
-   
+contains 
+
+   subroutine initialize_profiler(comm)
```

Comment:
Should be called `initialize()`  Use Fortran rename in the other layer to call it with the longer name.

Reply:
changed